addons/purchase_stock/models/product.py

```diff
@@ -75,6 +77,17 @@ def _compute_suggest_estimated_price(self):
             price = seller.price_discounted if seller else product.standard_price
             product.suggest_estimated_price = price * product.suggested_qty
 
+    def _search_product_with_suggested_quantity(self, operator, value):
```

Comment:
It's a shame I didn't see that in 19.0 but search methods are named as follow: _search_ + field name.
You can also move the method just below `_compute_suggested_quantity` (same field)
(Link to [guideline in documentation](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#symbols-and-conventions) about that point)
```suggestion
    def _search_suggested_quantity(self, operator, value):
```